test/mlir/accelerators/nnpa/conversion/lower-all-to-llvm-error-exit.mlir

```diff
@@ -0,0 +1,523 @@
+// RUN: onnx-mlir-opt --maccel=NNPA --convert-krnl-to-llvm="use-opaque-pointers=true func-call-error-exit=true"  %s -split-input-file | FileCheck %s
+
+// -----
+
+func.func @test_stick() -> () {
+  %0 = memref.alloc() : memref<10x10xf32>
+  %1 = memref.alloc() : memref<1x1x32x64xf32>
+  "zlow.stick"(%0, %1) : (memref<10x10xf32>, memref<1x1x32x64xf32>) -> ()
+  return
```

Comment:
The following tests are sharing the same pattern. It looks redundant to test for all operations.

Instead, you can have a test with two or three zlow ops. e.g.
```
y = zlow.stick(x)
z = zlow.relu(y)
out = zlow.unstick(z)
return out
```